Override @tootallnate/once to ^3.0.1.

@generated
@generator: scripts/apply-override.sh --patch common/override-tootallnate-once.diff --override 'global:@tootallnate/once=^3.0.1'
@override-package: @tootallnate/once@^3.0.1

Index: code-editor-src/package.json
===================================================================
--- code-editor-src.orig/package.json
+++ code-editor-src/package.json
@@ -238,6 +238,7 @@
   "overrides": {
     "braces": "^3.0.3",
     "postcss": "^8.4.31",
+    "@tootallnate/once": "^3.0.1",
     "node-gyp-build": "4.8.1",
     "serialize-javascript": "^7.0.3"
   },
